routing.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	health "github.com/Financial-Times/go-fthealth/v1_1"
+	"github.com/Financial-Times/message-queue-gonsumer/consumer"
+	status "github.com/Financial-Times/service-status-go/httphandlers"
+	"github.com/Sirupsen/logrus"
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	"net/http"
+	"github.com/Financial-Times/service-status-go/httphandlers"
+)
+
+type routing struct {
```

Comment:
I think the router knows too much about the rest of the app.

We could simply have it have a `HTTPMappingHandler` and `healthService` which are assembled in the main. This way the handler no longer holds on to a `*http.Client` or a `consumer.QueueConfig` which are actually dependencies of the health service